brainscore_language/benchmarks/syntaxgym/__init__.py

```diff
@@ -0,0 +1,44 @@
+from brainscore_core.benchmarks import BenchmarkBase
+from brainscore_core.metrics import Score
+from brainscore_language.artificial_subject import ArtificialSubject
+from brainscore_language import load_metric, benchmark_registry
+from brainscore_language.benchmarks.syntaxgym.sg_suite import _load_suite
+
+class SyntaxGymTSE(BenchmarkBase):
+    def __init__(self):
+        super(SyntaxGymTSE, self).__init__(
+            identifier='syntaxgym',
+            version=1,
+            parent='engineering',
+            ceiling=None,
+            bibtex=None)
+        self.metric = load_metric('accuracy')
+        self.data = _load_suite('brainscore_language/benchmarks/syntaxgym/test_suite.json')
```

Comment:
```suggestion
        self.data = _load_suite(Path(__file__).parent / 'test_suite.json')
```
should make this a bit more robust to being called/imported from different places.